torch_xla/amp/syncfree/sgd.py

```diff
@@ -0,0 +1,118 @@
+import torch
+import torch_xla
+from torch import Tensor
+from typing import List, Optional
+
+
+class SGD(torch.optim.SGD):
```

Comment:
Can you add the link to pytorch's SGD github link in the front of the comment since this code came from there.